src/index.ts

```diff
@@ -31,7 +31,7 @@ program
  * validate that the package.json file and
  * node_modules folder exists.
  */
-const opts = getCLIOptions(program.parse(process.argv));
+const opts = getCLIOptions(program.parse().opts());
```

Comment:
Refer to https://github.com/tj/commander.js/blob/v9.1.0/CHANGELOG.md#700-2021-01-15 for details about the breaking `commander` changes around option handling.

I also omitted the `process.argv` parameter because [it gets used by default by `parse()`](https://github.com/tj/commander.js/tree/v9.1.0#parse-and-parseasync).